src/locales/en.json

```diff
@@ -138,6 +138,7 @@
     "person_risk_profile_question": "What is their risk profile?",
     "pointsdisplay": {
       "empty_warning": "Fill in calculator to see risk level",
+      "jump_to_error": "Jump to error",
```

Comment:
Could we call this "Jump to missing data"

Or "Fill in $FIELD to see results (jump to $FIELD)"

Reply:
Hmm, I considered "Jump to missing data" and actually decided against it, I think it's more confusing (missing data sounds like it's missing on our end)

Reply:
Could we look to a UI pattern library and see how this is commonly done?

Reply:
Since @jy2wong often links to Nielsen, I will follow her lead: https://www.nngroup.com/articles/errors-forms-design-guidelines/

Also - https://designmodo.com/ux-form-validation/#4-error-messages

My takeaway from a (very) quick skim of these and a few other articles is basically: attempt to provide inline validation as soon as the error occurs. In my mind, this would mean if they fill out any field after the risk profile field before risk profile has been filled out, highlight it with an error message immediately.

If for some reason we don't go that route,, I wonder about not having any "Jump to error" message and instead just jumping them to it automatically by doing an autoscroll and show the error message then.

Reply:
IMO neither of those options are great - for the first one, they aren't going to be able to see that it's highlighted since they already scrolled past. For the second one - when do we autoscroll? Literally not letting them fill anything else out until they fill out the risk profile seems really annoying

Reply:
Oh right. There's no "submit" button. I see.... 


Well makes sense how you arrived at this solution, then! 🙃

Reply:
Personally I'd love it if you could seek out @jy2wong's take on this before publishing. But I don't think I'd block on it.